addons/context_actions/functions/fnc_healUnits.sqf

```diff
@@ -0,0 +1,36 @@
+/*
+ * Author: 3Mydlo3, veteran29
+ * Heals units in given selection. Works for units in vehicles.
+ * Healing modes:
+ * HEAL_MODE_ALL - 0
+ * HEAL_MODE_AI - 1
+ * HEAL_MODE_PLAYERS - 2
+ *
+ * Arguments:
+ * 0: Selected Objects <ARRAY>
+ * 1: Mode <SCALAR>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[player1, player2, player3], 2] call zen_context_actions_fnc_healUnits
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_objects", "_mode"];
+
+private _units = [];
+{
+    _units append crew _x;
+} forEach _objects;
+
+private _fnc_filter = [{true}, {!isPlayer _this}, {isPlayer _this}] select _mode;
```

Comment:
Can replace the `_this` with `_x` and then in the if condition just do `alive _x && _fnc_filter`.